### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by Hans5958** [ScratchAddons/ScratchAddons#539] [Discussion]
I need questions for https://scratchaddons.com/faq. Drop what questions do you think it will be asked frequently below. 

Updates can be seen on https://github.com/ScratchAddons/website/blob/faq/site/faq.md.

**Comment by retronbv** [comment]
Does this take my password?
Is this safe?
I want to help. How can I?

**Comment by j5155** [comment]
What do all the addons do?
What should I do if I find a problem?
Can I tell people on Scratch about this?
Are there any consequences for enabling everything?

**Comment by WorldLanguages** [comment]
Performance warnings probably deserve a FAQ

**Comment by GrahamSH-LLK** [comment]
This should probably be a discussion.

**Comment by retronbv** [comment]
Yeah probably

**Comment by Hans5958** [comment]
I was going to close this when I release the first public version of the FAQ, and then I will make another discussion post. But, you can make this to be a discussion if you want.

**Comment by Explosion-Scratch** [comment]
How can I help with the development?

**Reply by Explosion-Scratch** [reply]
That is a proposed question for the FAQ, not a literal one.

**Reply by Explosion-Scratch** [reply]
Why am I not allowed to discuss ScratchAddons on scratch?

**Reply by GrahamSH-LLK** [reply]
Good one!

**Comment by Explosion-Scratch** [comment]
Thanks!

**Comment by Hans5958** [comment]
Reminder: Updates can been seen on https://github.com/ScratchAddons/website/blob/faq/site/faq.md.
